.github/workflows/build.yml

```diff
@@ -113,6 +114,55 @@ jobs:
           message: |
             Build succeeded and deployed at http://${{ env.surge_url }} 
             (hash ${{ github.event.pull_request.head.sha }} deployed at ${{ steps.date.outputs.date }})
+
+  master_frontend_build_and_deploy:
+    needs: frontend_tests
+    if: github.event_name == 'push' && github.ref == 'refs/heads/master'
+    name: master_frontend_build_and_deploy
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        country: [mozambique, cambodia, rbd]
+    env:
+      surge_url: prism-master-${{ matrix.country }}.surge.sh
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Get yarn cache directory path
```

Comment:
Is this step needed? Maybe not needed in the other deploy either?

Reply:
Its output is used by the following step, it was added 5 months ago. Do you know why?